package/MDAnalysis/analysis/distances.py

```diff
@@ -42,6 +42,7 @@
            'contact_matrix', 'dist', 'between']
 
 import numpy as np
+import scipy.sparse
```

Comment:
I like the full imports much better if we only use a few things. Much easier to grep. And really not much more to write. Why not be explicit with where you get functions from? And it keeps the name space tidier.

(Sidenote: I particular dislike function imports – I think they are really bad because you have no idea where they come from when reading code. )